components/Cards/index.js

```diff
@@ -17,3 +17,113 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+let articles = [];
+let bootstrap = [];
+let javascript = [];
+let jquery = [];
+let node = [];
+let technology = [];
+let topicArray = [];
+
+
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+.then(response =>{
+    console.log(response.data.articles);
+    articles = response.data.articles;
+    console.log(articles);
+
+
+    // articles.forEach(item =>{
+    //     topicArray.push(item);
+    // })
+    // console.log(topicArray);
+    // articles = Array.from(response.data.articles);
+    // console.log(articles);
+    // for (let i=0;i<articles.length;i++){
+    //     console.log(articles[i]);
+    //     topicArray.push(articles[i]);
+    //   }
+    //   console.log(topicArray);
+    //   return topicArray;
+      
+})
+.then(response =>{
+    bootstrap = Array.from(articles.bootstrap);
```

Comment:
good hack but how would you google to find how to loop through an object or turtn an object into an array?

Reply:
I think I would've probably googled something along the lines of "iterating through nested objects" or 'iterate through key-values". In looking it up since then, I think I would have used Object.values or for (let index in articles), or something along those lines.